client/src/components/layout/footer/copywright.tsx

```diff
@@ -0,0 +1,9 @@
+"use client";
+
+export function Copyright() {
+  return (
+    <div className="font-sans text-sm leading-5 font-normal text-stone-500">
+      Copyright {new Date().getFullYear()} SCP
```

Comment:
Normally, the year in the copyright notice reflects the time the site was created, so I would keep it static.